demo/demo-jaxrs/jaxrs-server/src/main/java/org/apache/servicecomb/demo/jaxrs/server/RequestClientTimeOut.java

```diff
@@ -35,7 +35,7 @@
   @Path("/add")
   @POST
   public int add(@FormParam("a") int a, @FormParam("b") int b) throws InterruptedException {
-    Thread.sleep(1000);
+    Thread.sleep(3000);
```

Comment:
Do we need to sleep for 3 second?

Reply:
more than 2 seconds. I found time based test cases may not work property if we using time smaller than 1 seconds. So used 1 second as a unit. Integration tests takes more several seconds and I think is acceptable.